extensions/benchmark/src/main/java/ai/djl/benchmark/Arguments.java

```diff
@@ -204,13 +208,22 @@ static Options getOptions() {
                         .argName("NUMBER_THREADS")
                         .desc("Number of inference threads.")
                         .build());
-        options.addOption(
+        OptionGroup deviceGroup = new OptionGroup();
+        deviceGroup.addOption(
                 Option.builder("g")
                         .longOpt("gpus")
                         .hasArg()
                         .argName("NUMBER_GPUS")
                         .desc("Number of GPUS to run multithreading inference.")
                         .build());
+        deviceGroup.addOption(
+                Option.builder("nc")
+                        .longOpt("neuron-cores")
+                        .hasArg()
+                        .argName("NEURON-CORES")
+                        .desc("Number of neuron cores to run multithreading inference.")
```

Comment:
Can you describe what neuron cores are somewhere? or link to a reference on them

Reply:
I actually could not find a good document about neuron core. The only place I found is: https://awsdocs-neuron.readthedocs-hosted.com/en/latest/neuron-guide/neuron-runtime/nrt-configurable-parameters.html,

But it doesn't explain what is core

Reply:
Honestly, I would be fine with adding "See https://awsdocs-neuron.readthedocs-hosted.com/". It just needs something to indicate what this neuron core thing is about for those who have not heard the term before